taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -22,6 +22,19 @@ import { createBaseURL, toSnakeCase } from "@/utils";
 
 const branchNameRegEx = /[^a-z0-9_]+/;
 
+function formatDate(date) {
+  const map = {
+    mm: date.getMonth() + 1,
+    dd: date.getDate(),
+    yy: date.getFullYear().toString().slice(-2),
+    yyyy: date.getFullYear(),
+    HH: date.getHours().toString().slice(-2),
+    MinMin: date.getMinutes().toString().slice(-2),
+  };
+
+  return `${map.mm}${map.dd}${map.yy}_${map.HH}_${map.MinMin}`;
+}
+
```

Comment:
Why not use the built-in `Date` methods? https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString